test-case-reporting/app.ts

```diff
@@ -144,10 +146,26 @@ app.post('/report', jsonParser, async (req, res) => {
 app.get('/_cron/compute-stats', async (req, res) => {
   const {count} = req.query;
 
-  const message = `Computing pass/fail % for past ${count} runs`;
-  res.send(message);
+  try {
+    // This header is set by App Engine when running cron tasks, and is
+    // stripped out of external requests.
+    if (!req.header('X-Appengine-Cron') && process.env.NODE_ENV !== 'dev') {
+      throw new Error('Attempted external request to a cron endpoint');
+    }
+
+    const sampleSize = parseInt(count.toString());
```

Comment:
Prefer to use `Number(count.toString())` (I'm not even sure if you really need to call `.toString` in this case, you can test it out - `parseInt` is too forgiving of malformed number-strings

Reply:
`toString` is required by TypeScript since query params can potentially be arrays

Reply:
Done.

Reply:
TypeScript didn't complain about the ToString for the argument to Number() -- looks like it doesn't worry about the type as much.